verticapy/tests_new/core/vdataframe/test_typing.py

```diff
@@ -14,24 +14,110 @@
 See the  License for the specific  language governing
 permissions and limitations under the License.
 """
-import os
 from itertools import chain
+import os
+
 import pytest
+
 import verticapy as vp
-from verticapy.utilities import read_json
+from verticapy.errors import ConversionError
+from verticapy.utilities import read_json, TableSample
 
 
 class TestTyping:
```

Comment:
Again, i think we should follow the origianl file and have two separate classes for vDC and vDF

Reply:
done